pkg/cedar/intake/models/easi_action.go

```diff
@@ -1,7 +1,7 @@
 package models
 
 // NOTE: this type is used to create a schema used by the CEDAR Intake API
-// When changing this type, update the version for it in pkg/cedar/intake/translation/constants.go (IntakeInputSchemaEASIActionVersion)
+// When changing this type, add a new version for it in pkg/cedar/intake/translation/constants.go
```

Comment:
This change undoes work from https://github.com/CMSgov/easi-app/pull/1668; please manually reverse it.